server/src/internalClusterTest/java/org/opensearch/snapshots/SearchableSnapshotIT.java

```diff
@@ -332,6 +335,13 @@ private void assertIndexSettingChangeBlocked(String index) {
         }
     }
 
+    private void assertUpdateIndexSettingsAllowList(String index) {
```

Comment:
Can we add a case for a request that contains both an allowed setting and a disallowed setting?